lib/temporal/metadata/activity.rb

```diff
@@ -23,6 +23,21 @@ def initialize(namespace:, id:, name:, task_token:, attempt:, workflow_run_id:,
       def activity?
         true
       end
+
+      def to_h
+        {
+          'attempt' => attempt,
+          'headers' => headers,
+          'heartbeat_details' => heartbeat_details,
+          'activity_id' => id,
+          'activity_name' => name,
+          'namespace' => namespace,
+          'task_token' => task_token,
```

Comment:
I don't think this is useful to anyone

Reply:
We actually just used this as part of the information required to debug the cron workflow issue we were discussing on slack. We noticed the task token was a query token, not a workflow task token. Granted I'm not sure how useful it would normally be.